roles/openshift_facts/library/openshift_facts.py

```diff
@@ -59,11 +59,12 @@ def migrate_docker_facts(facts):
             'additional_registries',
             'insecure_registries',
             'blocked_registries',
-            'options'
+            'options',
+            'use_system_container',
         ),
         'node': (
             'log_driver',
-            'log_options'
+            'log_options',
```

Comment:
Leftover from when I updated the section but removed since then. I'll remove it. It's aesthetics only.